src/backend/src/services/projects.services.ts

```diff
@@ -627,4 +629,20 @@ export default class ProjectsService {
 
     return newManufacturer;
   }
+
+  /**
+   * Get all the manufacturers in the database.
+   * @returns all the manufacturers
+   */
+  static async getAllManufacturers(submitter: User): Promise<Manufacturer[]> {
+    if (submitter.role === Role.GUEST) {
+      throw new AccessDeniedGuestException('Guests cannot get the manufacturers');
```

Comment:
<img width="326" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/84bb7b9a-58d3-463f-a709-6a39c65e1aab">
error message looks like this rn, this helper method is honestly kinda weird, can you just use the AccessDeniedException or clean up how this helper method prints out the error message